js/function.js

```diff
@@ -0,0 +1,53 @@
+// Функция для проверки длины строки
+
+const searchStringValidation = (test, number) => {
+  const lenght = test.length >= number;
+  return lenght;
+};
+
+searchStringValidation('кот', 3);
+
+// Функция для проверки, является ли строка палиндромом.
+
+const checkPalindrom = (string) => {
+  let newString = string.toLowerCase().replaceAll(' ', '');
+
+  for (let i = string.length; i >= 0; i--) {
+    newString += string[i];
+  }
+  return newString === string;
```

Comment:
return следует отделять пустой строкой
```suggestion
  for (let i = string.length; i >= 0; i--) {
    newString += string[i];
  }
  
  return newString === string;
```